client/internal/config.go

```diff
@@ -88,6 +90,12 @@ func ReadConfig(configPath string) (*Config, error) {
 		if _, err := util.ReadJson(configPath, config); err != nil {
 			return nil, err
 		}
+
+		// Should be redondant with createEngineConfig declaration on connect.go, but easier to read
+		if config.ServerSSHAllowed == nil {
+			config.ServerSSHAllowed = util.True()
+		}
```

Comment:
We need to move this from here to `update`. why? This will only update the memory-loaded configuration, but the client configuration file will not be updated. See the comment on the update function

Reply:
Understood, done in the last commit